.github/workflows/vale.yaml

```diff
@@ -0,0 +1,16 @@
+name: Vale
+on: [pull_request]
+
+jobs:
+  vale:
+    name: runner / vale
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: errata-ai/vale-action@reviewdog
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+        with:
+          files: "content/docs"
```

Comment:
Is it possible to run it only on the files that are changed? Or even just on the changes? Wait I guess that's what it's doing now right?